### DISCUSSION TITLE
Mobile flasher

### DISCUSSION THREAD

**Discussion by pauljbernhardt-stack** [community/community#28149] [Other Feature Feedback, Questions, & Ideas]
A business startup flashing automobile computers mobility

**Comment by pauljbernhardt-stack** [comment]
Most mechanics use a diagnostic computer to identify what part of the automobile is malfunctioned did not show time it’s the computer has malfunctioned if low voltage gets to the computer it forgets like anything else the computer then has to be soon away to be reprogrammed just causes a lot of downtime so I was going to do it Mobley for big businesses people the only have one card get back-and-forth to work

**Comment by jfullstackdev** [comment]
what do you want to achieve ? so we can help

**Reply by pauljbernhardt-stack** [reply]
As a double amputee below the knees  and master electrician I still want to work teach what I know and learn the computer world better I walk on blades I never give up I love to inspire others to never stop even when everything is against you
